frontend/src/scenes/embedded-analytics/query-endpoints/QueryEndpoints.tsx

```diff
@@ -0,0 +1,199 @@
+import { dayjs } from 'lib/dayjs'
+
+import { QueryEndpointType } from '~/types'
+
+import { queryEndpointsLogic } from './queryEndpointsLogic'
+import { queryEndpointLogic } from 'scenes/data-warehouse/editor/sidebar/queryEndpointLogic'
+
+const SQLButton = ({ sql }: { sql: string }): JSX.Element => {
+    const [popoverVisible, setPopoverVisible] = useState(false)
+
+    return (
+        <Popover
+            visible={popoverVisible}
+            onMouseEnterInside={() => setPopoverVisible(true)}
+            onMouseLeaveInside={() => setPopoverVisible(false)}
+            overlay={
+                <div className="p-2 max-w-md">
+                    <div className="text-sm font-medium mb-2">SQL Query</div>
+                    <pre className="text-xs bg-gray-100 p-2 rounded overflow-x-auto whitespace-pre-wrap">{sql}</pre>
+                </div>
+            }
+            placement="top"
+        >
+            <div onMouseEnter={() => setPopoverVisible(true)} onMouseLeave={() => setPopoverVisible(false)}>
+                <LemonButton type="secondary" size="small" icon={<IconCode />} />
+            </div>
+        </Popover>
+    )
+}
+
+export function QueryEndpoints(): JSX.Element {
+    return (
+        <>
+            <QueryEndpointsTable />
+        </>
+    )
+}
+
+export const QueryEndpointsTable = (): JSX.Element => {
+    const { setFilters } = useActions(queryEndpointsLogic)
+    const { filters } = useValues(queryEndpointsLogic)
+    const { queryEndpoints, queryEndpointsLoading } = useValues(queryEndpointsLogic)
+    const { deleteQueryEndpoint } = useActions(queryEndpointLogic)
+
+    const columns: LemonTableColumns<QueryEndpointType> = [
+        {
+            title: 'Name',
+            key: 'name',
+            dataIndex: 'name',
+        },
+        {
+            title: 'Description',
+            key: 'description',
+            dataIndex: 'description',
+        },
+        {
+            title: 'Status',
+            key: 'is_active',
+            dataIndex: 'is_active',
+            render: (_, record) => (
+                <span>
+                    {record.is_active ? (
+                        <LemonTag type="success">Active</LemonTag>
+                    ) : (
+                        <LemonTag type="danger">Inactive</LemonTag>
+                    )}
+                </span>
+            ),
+        },
+        {
+            title: 'Created At',
+            key: 'created_at',
+            dataIndex: 'created_at',
+            render: (_, record) => new Date(record.created_at).toLocaleDateString(),
+        },
+        {
+            title: 'Created By',
+            key: 'created_by',
+            dataIndex: 'created_by',
+            render: (_, record) =>
+                record.created_by
+                    ? `${record.created_by.first_name} ${record.created_by.last_name}`.trim() || record.created_by.email
+                    : 'Unknown',
+        },
+        {
+            title: 'Endpoint Path',
```

Comment:
Sentence casing over title casing everywhere please (a handful of offenders in here to update)